snap/info.go

```diff
@@ -249,10 +249,25 @@ func (app *AppInfo) WrapperPath() string {
 	return filepath.Join(dirs.SnapBinariesDir, binName)
 }
 
+func (app *AppInfo) invocation(command string) string {
+	securityTag := app.SecurityTag()
+	return fmt.Sprintf("/usr/bin/ubuntu-core-launcher %s %s %s", securityTag, securityTag, filepath.Join(app.Snap.MountDir(), command))
```

Comment:
the systemd units seems to want the full abs path, it seems sane/safe in general